server/src/main/java/com/defold/extender/services/CocoaPodsService.java

```diff
@@ -739,13 +804,19 @@ private PodSpec createPodSpec(JSONObject specJson, PodSpec parent, File podsDir,
 
         // parse subspecs
         // https://guides.cocoapods.org/syntax/podspec.html#subspec
+        JSONArray defaultSubspecs = getAsJSONArray(specJson, "default_subspecs");
```

Comment:
Make sure to respect the default_subspec option and only use the subspec(s) specified in this option.